app/views/signin.js

```diff
@@ -27,12 +26,13 @@ define(['app', 'authentication'], function() {
         $http.post('/api/v2013/authentication/token', credentials).then(function onsuccess(success) {
 
             apiToken.set(success.data.authenticationToken);
-            
-            if($scope.rememberMe) localStorage.setItem('cbd_authentication_email', $scope.email);
-            else                  localStorage.removeItem('cbd_authentication_email');
 
-            if(success.data.expiration) localStorage.setItem("cbd_authentication_expiration", success.data.expiration);
-            else      localStorage.removeItem("cbd_authentication_expiration");
+            var expires = new Date();
+
+            expires.setDate(expires.getDate()+365);
+            
+            if($scope.rememberMe) $cookies.put   ('email', $scope.email, { path:'/', expires: expires, samesite: 'None', secure: true });
+            else                  $cookies.remove('email', { path:'/' });
```

Comment:
can you all add the expiration date to the cookie?
```
if(success.data.expiration) 
   $cookies.put("expiration", success.data.expiration, { path:'/', expires: new Date(success.data.expiration), samesite: 'None', secure: true });
else      
  $cookies.remove('expiration', { path:'/' });
            ```

Reply:
Done